staging/src/k8s.io/api/scheduling/v1alpha1/types.go

```diff
@@ -49,6 +49,11 @@ type PriorityClass struct {
 	// when this priority class should be used.
 	// +optional
 	Description string `json:"description,omitempty" protobuf:"bytes,4,opt,name=description"`
+
+	// nonPreempting specifies whether a pod with this PriorityClass could trigger a preemption process.
+	// If this field is unset, the PriorityClass is considered a preempting class by default.
+	// +optional
+	NonPreempting bool `json:"nonPreempting,omitempty" protobuf:"bytes,5,opt,name=nonPreempting"`
```

Comment:
do we still need to support v1alpha1 for this feature?

Reply:
also remove the field  NonPreempting  from  pkg/apis/scheduling/types.go?